### PR TITLE
(DOCS-14782): Add missing step

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCS-14782

### Staged Changes (Requires MongoDB Corp SSO)

- iOS Swift Tutorial: [Specify the Realm to Open](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCS-14782/tutorial/ios-swift/#specify-the-realm-to-open) (New step)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/tutorial/generated/swift-ios/local/ProjectsViewController.codeblock.user-in-realm-notification.swift [hunk 1]
```diff
@@ -1,2 +1,2 @@
-// TODO: fetch user data object
+self.userRealm = try! Realm(configuration: userRealmConfiguration)
```
Comment (nathan-contino-mongo) [inline source/tutorial/generated/swift-ios/local/ProjectsViewController.codeblock.user-in-realm-notification.swift:1]: Hm, are we missing the change in the source file for this bluehawk-generated codeblock? Or did we just forget to generate this when we updated the source?

File: source/includes/steps-tutorial-ios-swift-local.yaml [hunk 2]
```diff
@@ -84,6 +84,18 @@ content: |
      To learn more about how {+service-short+} Object models are used in iOS applications, 
      see :ref:`ios-read-and-write-data` in our iOS client guide.
 
+---
+title: Specify the Realm to Open
```
Comment (nathan-contino-mongo) [inline source/includes/steps-tutorial-ios-swift-local.yaml:88]: titlecase really muddies the waters on realm vs Realm :(

File: source/includes/steps-tutorial-ios-swift-local.yaml [hunk 3]
```diff
@@ -84,6 +84,18 @@ content: |
...
+ref: specify-the-realm-to-open
+content: |
+  Navigate to the ProjectsViewController.swift file. For the local application,
+  the user only has their own project - and their own Realm - to open.
```
Comment (nathan-contino-mongo) [inline source/includes/steps-tutorial-ios-swift-local.yaml:92]:
```suggestion
  the user only has their own project - and their own {+realm+} - to open. 
```

File: source/includes/steps-tutorial-ios-swift-local.yaml [hunk 4]
```diff
@@ -84,6 +84,18 @@ content: |
...
      see :ref:`ios-read-and-write-data` in our iOS client guide.
 
+---
+title: Specify the Realm to Open
+ref: specify-the-realm-to-open
+content: |
+  Navigate to the ProjectsViewController.swift file. For the local application,
+  the user only has their own project - and their own Realm - to open. 
+  We'll specify the :ref:`configuration <ios-configure-and-open-a-realm>` to use 
+  when we open the Realm in ``init(userRealmConfiguration: Realm.Configuration)``:
```
Comment (nathan-contino-mongo) [inline source/includes/steps-tutorial-ios-swift-local.yaml:94]:
```suggestion
  when we open the {+realm+} in ``init(userRealmConfiguration: Realm.Configuration)``:
```

File: source/includes/steps-tutorial-ios-swift-local.yaml [hunk 5]
```diff
@@ -60,7 +60,7 @@ content: |
        - Create the list view for the tasks in a given project.
 
      * - ProjectsViewController.swift
-       - Part 2: Display the list of projects where the logged
```
Comment (nathan-contino-mongo) [inline source/includes/steps-tutorial-ios-swift-local.yaml:63]: Would it make sense to add a "(in part 2, with sync)" or something to that effect so users aren't confused when this doesn't exist in the local-only version?

**Review by nathan-contino-mongo — APPROVED** [review]
LGTM with some source constants, a question, and a suggestion.

File: source/tutorial/generated/swift-ios/local/ProjectsViewController.codeblock.user-in-realm-notification.swift (hunk 1 above)
Comment (dacharyc) [reply source/tutorial/generated/swift-ios/local/ProjectsViewController.codeblock.user-in-realm-notification.swift:1]: I'm not sure how this got missed, honestly. I re-generated the `local` state to add this snippet, but before that, the snippet that was in the `local` generated files matched the `start` state. Not sure how that happened.

I do vaguely recall something about a bug related to this, I think... now I'm wondering if I've fiddled with this before and missed something. Let me do a bit more digging in git.

File: source/includes/steps-tutorial-ios-swift-local.yaml (hunk 5 above)
Comment (dacharyc) [reply source/includes/steps-tutorial-ios-swift-local.yaml:63]: How about: 
> Display the user's list of projects. In the local app, there is only the local user's project. In Part 2, when we add Sync, we'll add the projects where the logged in user is a member.

**Pull request merged**